curriculum/challenges/english/02-javascript-algorithms-and-data-structures/functional-programming/use-the-reduce-method-to-analyze-data.md

```diff
@@ -42,9 +44,11 @@ const usersObj = users.reduce((obj, user) => {
   obj[user.name] = user.age;
   return obj;
 }, {});
-console.log(usersObj); // { John: 34, Amy: 20, camperCat: 10 }
+console.log(usersObj);
 ```
 
+The console would display the value `{ John: 34, Amy: 20, camperCat: 10 }`.
+
 # --instructions--
 
 The variable `watchList` holds an array of objects with information on several movies. Use `reduce` to find the average IMDB rating of the movies **directed by Christopher Nolan**. Recall from prior challenges how to `filter` data and `map` over it to pull what you need. You may need to create other variables, and return the average rating from `getRating` function. Note that the rating values are saved as strings in the object and need to be converted into numbers before they are used in any mathematical operations.
```

Comment:
Do we want `directed by Christopher Nolan` translated?

Reply:
Maybe `Christopher Nolan` should be in code blocks and get rid of the strong tags.

Reply:
Good idea.